src/ape/managers/chain.py

```diff
@@ -975,71 +980,100 @@ def get(
             address (AddressType): The address of the contract.
             default (Optional[ContractType]): A default contract when none is found.
               Defaults to ``None``.
+            cache_to_disk (bool): Cache the fetched contract type to  (live networks only).
```

Comment:
```suggestion
            cache_to_disk (bool): Cache the fetched contract type to disk (live networks only).
```

Reply:
Of course! I'll review some old PRs to see how y'all deal with tests and make updates to the PR 👌